server/controllers/request.js

```diff
@@ -0,0 +1,105 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /requests
+// @desc load requests for logged in user
+// @access Private
+exports.loadRequests = asyncHandler(async (req, res, next) => {
+  const requests = await Request.find({ userId: req.user.id });
+
+  if (!requests) {
```

Comment:
The mongoose `find()` query returns an empty list if no profiles are found, to remain consistent with the response format it is recommended we return the empty list to the FE (where we can conditionally render a message if no profiles are found)